src/main/java/com/github/javafaker/service/FakeValuesService.java

```diff
@@ -153,10 +153,13 @@ public String bothify(String string, boolean isUpper) {
     /**
      * Generates a String that matches the given regular expression.
      */
```

Comment:
are you fixing a defect or adding a  new feature?  These two changes probably should be two different pull requests.  The maintainers may accept the bug fix but not the new feature.